app/api/connect/google/callback/route.ts

```
@@ -10,12 +10,12 @@ export async function GET(request: Request) {
   const code = searchParams.get("code");
   const state = searchParams.get("state");
 
-  if (!code || !state) return NextResponse.redirect(`${getBaseUrl()}/mailboxes/${state}/settings?error=invalid_code`);
+  if (!code || !state) return NextResponse.redirect(`${getBaseUrl()}/mailboxes/${state}/settings/integrations?error=invalid_code`);
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)